src/App.js

```diff
@@ -2,9 +2,12 @@ import { Route, Routes } from "react-router-dom";
 import "./App.css";
 import Header from "./components/Header/Header";
 import SubHeader from "./components/SubHeader/SubHeader";
+
 import TopBar from "./components/TopBar/TopBar";
+import CartPage from "./pages/CartPage/CartPage";
 import HomePage from "./pages/HomePage/HomePage";
 import ProductListingPage from "./pages/ProductListingPage/ProductListingPage";
+import Signin from "./pages/Signin/Signin";
```

Comment:
You can create an index file in the pages folder and export all the components in the folder from this index file and then import from the pages folder only.

It makes the code cleaner. 

Refer to the Filter.jsx file in this PR 

https://github.com/junaidshaikh-js/autodecore-react/pull/9/files

Reply:
Thanks @junaidshaikh-js for the suggestion and for sharing the reference. I am planning to refactor the folder and import structure soon.